src/HomeKit/HMEnums.cs

```diff
@@ -492,8 +496,8 @@ public enum HMCharacteristicType {
 		NitrogenDioxideDensity,
 
 		[iOS (10,2), Watch (3,1,1), TV (10,1)]
-		[Field ("HMCharacteristicTypeSuplhurDioxideDensity")]
-		SuplhurDioxideDensity,
+		[Field ("HMCharacteristicTypeSulphurDioxideDensity")]
+		SulphurDioxideDensity,
```

Comment:
The original value here probably required a fix to the typo test for `Suplhur`, can you check that (and revert it if it's the case)?